dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp

```diff
@@ -693,14 +693,14 @@ ModelBus::evalZ(const double& /*t*/) {
     topologyModified_ = true;
     if (currState == OPEN) {
       switchOff();
-      network_->addEvent(id_, DYNTimeline(NodeOff));
+      if (network_->hasTimeline()) network_->addEvent(id_, DYNTimeline(NodeOff));
```

Comment:
useless tests

Reply:
That is to prevent building a DYNTimeline(...) object. It is exactly the same as
if ( network_->hasTimeline()) {
      DYNTimeline_object TL( ....);
      network_->addEvent( id_, TL) ;
}
It is exactly what is asked in the preceding lines with MessageTimeline. I don't know what to do

Reply:
please see my proposal above

Reply:
ok, I close resolve this discussion since it points to anotherone